### ISSUE TITLE
Installation warning: maintainer-address-missing shlomi-noach

### ISSUE THREAD

**Issue by bancer** [github/gh-ost#708]
I downloaded and executed gh-ost_1.0.47_amd64.deb file in Ubuntu 14.04 LTS. During the installation this warning appeared:

```
The installation of a package which violates the quality standards isn't allowed. 
This could cause serious problems on your computer. Please contact the person 
or organisation who provided this package file and include the details beneath.

Lintian check results for /...my-path.../gh-ost_1.0.47_amd64.deb:
E: gh-ost: maintainer-address-missing shlomi-noach

**Comment by shlomi-noach** [comment]
Thank you, I'll look into it.

**Comment by shlomi-noach** [comment]
closed by #710
